proxy/pom.xml

```diff
@@ -107,6 +102,16 @@
       <groupId>com.google.code.findbugs</groupId>
       <artifactId>jsr305</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-all</artifactId>
+      <version>4.0.33.Final</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-daemon</groupId>
+      <artifactId>commons-daemon</artifactId>
+      <version>1.0.15</version>
```

Comment:
Done -- since we're using a Dockerfile the build is fairly straightforward no matter how we decide to organize things. PTAL.